src/main/java/fi/otavanopisto/kuntaapi/server/integrations/tpt/client/model/ResponseHeader.java

```diff
@@ -0,0 +1,55 @@
+package fi.otavanopisto.kuntaapi.server.integrations.tpt.client.model;
+
+import java.io.Serializable;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ResponseHeader implements Serializable {
+
+  private static final long serialVersionUID = 6432250874166877639L;
+
+  @JsonProperty("qtime")
+  private Integer qtime;
+
+  @JsonProperty("QTime")
+  private Integer QTime2;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "QTime2" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)